sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-Demand.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Please only use present tense (will be used by -&gt; are used by). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Doesn't seem like this was addressed. Btw, there are multiple "will be used by" in this section</p>

Reply:
<p>Just curious, can you see this comment?</p>

Reply:
<p>Must address before merge!</p>

Reply:
ack

Reply:
done